core/layers/svc-core-proxy/extension/main.go

```
@@ -0,0 +1,114 @@
+	if err != nil {
+		log.Fatalf("failed to load aws configuratio %+v", err)
+	}
+
+	err2 := startMain(configuration)
+	if err2 != nil {
+		log.Fatalf("failed to start main %+v", err2)
+	}
+}
+
+func startMain(configuration *clients.Configuration) error {
+	ctx, cancel := context.WithCancel(context.Background())
```

golangci-lint (1.62.2)

lostcancel: the cancel function is not used on all paths (possible context leak)

(govet)